### DISCUSSION TITLE
Using scripts with the plugin

### DISCUSSION THREAD

**Discussion by NFTYieldFarm** [jeffreylanters/react-unity-webgl#393] [Question and Answer]
Hello, I have built in socket-io to my unity project and need to expose the webGL instantiation to it.

In normal html I just wrap the var with the script and it works well:

<script src="TemplateData/UnityProgress.js"></script>
    <script src="/socket.io/socket.io.js"></script>
    <script src="Build/UnityLoader.js"></script>
    <script>
      var unityInstance = UnityLoader.instantiate("unityContainer", "Build/public.json", {onProgress: UnityProgress});
    </script>

but how would I do this with the plugin?

**Comment by jeffreylanters** [comment]
Hi! Where most plugins add these scripts to your build's HTML file, you should be able to copy these scripts, and add these manually to the React's HTML file and public directory.
